cmd/tcpgoon.go

```diff
@@ -63,6 +65,13 @@ func validateRequiredArgs(params *tcpgoonParams, args []string) error {
 		return errors.New("Number of required parameters doesn't match")
 	}
 	params.hostPtr = args[0]
+
+	addrs, err := net.LookupIP(params.hostPtr)
```

Comment:
what if we already got an IP address?

Reply:
It's solved by the LookupIP  (https://golang.org/src/net/lookup.go#L177)